<!DOCTYPE html>
<html>

<head>
  <title>Curriculum Vitae</title>
  <link rel="stylesheet" href="https://use.typekit.net/vty3ovx.css">
  <link rel="stylesheet" href="styles.css">
  <script src="cv.js"></script>
</head>

<body>
  <table border=0 cellspacing=0 cellpadding=0>
    <tr class="caption">
      <td colspan=2 valign=top>
        <p><span class="dark">Curriculum</span>&emsp;<span class="light">Vitae</span></p>
      </td>
    </tr>
    <tr><td width=25%></td><td width=75%></td></tr>
    <tr class="section title">
      <td colspan=2 valign=top>
        <p><span>Personal Data</span></p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>Name</span></p>
      </td>
      <td class="value" valign=top>
        <p><span>David Luis Wiegandt</span></p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>Born</span></p>
      </td>
      <td class="value" valign=top>
        <p><span>16.09.1993 in Berlin</span></p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>GitHub</span></p>
      </td>
      <td class="value" valign=top>
        <p><span><a href="https://github.com/dlw93">https://github.com/dlw93</a></span></p>
      </td>
    </tr>
    <tr class="section title">
      <td colspan=2 valign=top>
        <p><span>Education</span></p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>10/2016 &mdash; present</span></p>
      </td>
      <td class="value" valign=top>
        <p>
          <span>Master's Studies of Computer Science, Humboldt-Universität zu Berlin</span>
          <br />
          <span><em>Current average grade of 1.1</em></span>
        </p>
        <p>
          <span><em>Thesis: &ldquo;Distributed Instance-Optimal Join Processing&rdquo; (in progress)</em></span>
        </p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>10/2012 &mdash; 06/2016</span></p>
      </td>
      <td class="value" valign=top>
        <p>
          <span>Bachelor's Studies of Computer Science, Humboldt-Universität zu Berlin</span>
          <br />
          <span><em>Bachelor of Science (2.2)</em></span>
        </p>
        <p><span><em>Thesis: &ldquo;Local Graph Patterns for Scientific Workflow Similarity Search&rdquo; (graded
              1.0)</em></span>
        </p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>08/2004 &mdash; 06/2012</span></p>
      </td>
      <td class="value" valign=top>
        <p>
          <span>Primo-Levi-Gymnasium, Berlin</span>
          <br />
          <span><em>Abitur (2.2)</em></span>
        </p>
      </td>
    </tr>
    <tr class="section title">
      <td colspan=2 valign=top>
        <p><span>Work Experience</span></p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>04/2020 &mdash; 03/2021</span></p>
        </td>
        <td class="value" valign=top>
          <p>
            <span><a href="https://neurologie.charite.de/forschung/arbeitsgruppen/bewegungsstoerungen_andrea_kuehn/">Department of
                Neurology, Charité &mdash; Universitätsmedizin Berlin</a></span>
            <br />
            <span><em>Automated report generation for outpatient deep brain stimulation visits and development of a web app for
                domain-specific patient information management</em></span>
          </p>
        </td>
        </tr>
        <tr>
          <td class="key" valign=top>
              <p><span>06/2015 &mdash; 12/2019</span></p>
            </td>
            <td class="value" valign=top>
              <p>
            <span><a href="https://www.informatik.hu-berlin.de/de/forschung/gebiete/wbi">Knowledge Management in Bioinformatics,
                Humboldt-Universität zu Berlin</a></span>
            <br />
            <span><em>Collaboration in various research projects and work as a tutor in different courses at the chair
                of
                &ldquo;Knowledge Management in Bioinformatics&rdquo; led by Prof. Ulf Leser</em></span>
          </p>
        </td>
        </tr>
        <tr>
          <td class="key" valign=top>
        <p><span>08/2016 &mdash; 11/2016</span></p>
      </td>
      <td class="value" valign=top>
        <p>
          <span><a href="https://www.psimetals.de/">PSI Metals GmbH, Berlin</a></span>
          <br />
          <span><em>Implementation of an automated release history update in the internal wiki upon a new software
              release</em></span>
        </p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>01/2016 &mdash; 07/2016</span></p>
      </td>
      <td class="value" valign=top>
        <p>
          <span><a href="https://www.interactive-tools.de/">interactive tools GmbH, Berlin</a></span>
          <br />
          <span><em>Enhancements to the CI/CD workflows and participation in web development projects</em></span>
        </p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>10/2013 &mdash; 05/2015</span></p>
      </td>
      <td class="value" valign=top>
        <p>
          <span><a href="https://www.psimetals.de/">PSI Metals GmbH, Berlin</a></span>
          <br />
          <span><em>Automation and parametrisation of various build, installation and test workflows and maintenance of
              the
              internal GUI
              framework</em></span>
        </p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>02/2013 &mdash; 06/2013</span></p>
      </td>
      <td class="value" valign=top>
        <p>
          <span><a href="https://kantiko.com/">kantiko GmbH, Berlin</a></span>
          <br />
          <span><em>Collaboration in the development of a web application providing small and medium-sized businesses
              with a
              simple means to
              perform their accounting</em></span>
        </p>
      </td>
    </tr>
    <tr class="section title">
      <td colspan=2 valign=top>
        <p><span>Languages</span></p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>Natural</span></p>
      </td>
      <td class="value" valign=top>
        <p>
          <span>German (native speaker), English (fluent in spoken and written)</span>
        </p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>Programming</span></p>
      </td>
      <td class="value" valign=top>
        <p>
          <span>TypeScript/JavaScript, HTML, CSS, Java, Erlang, Python, SQL, C, C#, &hellip;</span>
        </p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>Technologies</span></p>
      </td>
      <td class="value" valign=top>
        <p>
          <span>Node.js, Angular, React, Lucene, Hibernate, &hellip;</span>
        </p>
      </td>
    </tr>
    <tr class="section title">
      <td colspan=2 valign=top>
        <p><span>Publications</span></p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>Author</span></p>
      </td>
      <td class="value" valign=top>
        <p>
          <span>Wiegandt, David Luis, et al.</span>
          <span><a
              href="https://www.semanticscholar.org/paper/Graph-n-grams-for-Scientific-Workflow-Similarity-Wiegandt-Starlinger/c4dd57f4b15e846ff98b6574dcbb682dbd7bd9cf"><em>&ldquo;Graph
                n-grams for Scientific Workflow Similarity Search.&rdquo;</em> LWDA. 2016.</a></span>
        </p>
      </td>
    </tr>
    <tr>
      <td class="key" valign=top>
        <p><span>Co-Author</span></p>
      </td>
      <td class="value" valign=top>
        <p>
          <span><a href="https://www.sciencedirect.com/science/article/abs/pii/S2214579620300010">Habibi, Maryam, et al.
              <em>&ldquo;PatSeg: A Sequential Patent Segmentation Approach.&rdquo;</em>
              Big Data Research 19-20 (2020): 100133.</a></span>
          </p>
          <p>
          <span><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2958-3">Ševa, Jurica, et al.
              <em>&ldquo;VIST - A Variant-Information Search Tool for precisiononcology.&rdquo;</em>
              BMC bioinformatics 20.1 (2019): 429.</a></span>
        </p>
        <p>
          <span><a href="https://ascopubs.org/doi/10.1200/PO.18.00371">Pallarz, Steffen, et al. <em>&ldquo;Comparative Analysis of
                Public Knowledge Bases for Precision
              Oncology.&rdquo;</em>
            JCO Precision Oncology 3 (2019): 1-8.</a></span>
        </p>
        <p>
          <span><a href="https://pubmed.ncbi.nlm.nih.gov/28881963/">Habibi, Maryam, et al.
              <em>&ldquo;Deep learning with word embeddings improves biomedical named entity
              recognition.&rdquo;</em>
            Bioinformatics 33.14 (2017): i37-i48.</a></span>
        </p>
        <p>
          <span><a
              href="https://www.semanticscholar.org/paper/Recognizing-chemicals-in-patents%3A-a-comparative-Habibi-Wiegandt/18ec62911381f913918711ac57dcdc4fd9f5845c">Habibi,
              Maryam, et al. <em>&ldquo;Recognizing chemicals in patents: a comparative analysis.&rdquo;</em>
              Journal of cheminformatics 8.1 (2016): 59.</a></span>
        </p>
        <p>
          <span><a
              href="https://www.semanticscholar.org/paper/Performance-of-Gene-Name-Recognition-Tools-on-Habibi-Wiegandt/06b2d2ab672148bd60685d942ab8b2dded2558ea">Habibi,
              Maryam, et al. <em>&ldquo;Performance of Gene Name Recognition Tools on Patents.&rdquo;</em>
              SMBM. 2016.</a></span>
        </p>
      </td>
    </tr>
  </table>
</body>

</html>
